src/app/forms/templates/psv/psv-tech-record.template.ts

```diff
@@ -119,91 +142,115 @@ export const PsvTechRecord: FormNode = {
+      editType: FormNodeEditTypes.SELECT,
+      options: getOptionsFromEnum(EuVehicleCategories),
+      validators: [{ name: ValidatorNames.Required }]
     },
     {
       name: 'emissionsLimit',
       label: 'Emission limit (plate value)',
       value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }]
     },
     {
-      name: 'seatsLowerDeck',
-      label: 'Seats lower deck',
+      name: 'seatsUpperDeck',
+      label: 'Seats Upper deck',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }, { name: ValidatorNames.Required }],
+      id: 'style-float-right-small'
     },
     {
-      name: 'seatsUpperDeck',
-      label: 'Seats upper deck',
+      name: 'seatsLowerDeck',
+      label: 'Seats Lower deck',
       value: '',
-
+      width: FormNodeWidth.XS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 3 }, { name: ValidatorNames.Required }]
     },
     {
       name: 'standingCapacity',
       label: 'Standing capacity',
       value: '',
-
-      type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
-    },
-    {
-      name: 'vehicleSize',
-      label: 'Vehicle size',
-      value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 3 }, { name: ValidatorNames.Required }]
     },
     {
       name: 'numberOfSeatbelts',
       label: 'Number of seat belts',
       value: '',
-
+      width: FormNodeWidth.XXS,
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.TEXT,
+      validators: [{ name: ValidatorNames.MaxLength, args: 2 }, { name: ValidatorNames.Required }]
     },
+
     {
       name: 'seatbeltInstallationApprovalDate',
       label: 'Seatbelt installation approval date / type approved',
       value: '',
-
       type: FormNodeTypes.CONTROL,
-      viewType: FormNodeViewTypes.STRING
+      viewType: FormNodeViewTypes.STRING,
+      editType: FormNodeEditTypes.DATE,
+      isoDate: false
+    },
+    {
+      name: 'vehicleSize',
+      label: 'Vehicle size',
+      value: '',
+      type: FormNodeTypes.CONTROL,
+      viewType: FormNodeViewTypes.STRING,
```

Comment:
Don't need to declare this on any that is a string, it defaults to this I believe